history-service/index.js

```diff
@@ -0,0 +1,35 @@
+import express from 'express'
+import cors from 'cors'
+import mongoose from 'mongoose'
+import 'dotenv/config'
+import historyRoutes from "./routes/historyRoutes.js"
+
+let uri;
+
+if (process.env.ENV == "PROD") {
```

Comment:
To discuss on standardization whether to use `ENV` or `NODE_ENV` next week. If we use `NODE_ENV` then the two possible values would be `development` and `production` by convention.

https://stackoverflow.com/questions/16978256/what-is-node-env-and-how-to-use-it-in-express